docs/xmlprofiles.rst

```diff
@@ -737,39 +737,43 @@ This section of the :class:`Participant's rtps` configuration allows defining bu
-+---------------------------+---------------------------------------+-------------------------+-----------------------+
-| |readhistmem|             | Memory policy for builtin             | |mempol|                | :class:`PREALLOCATED` |
-|                           | readers.                              |                         |                       |
-|                           |                                       |                         |                       |
-+---------------------------+---------------------------------------+-------------------------+-----------------------+
-| |writhistmem|             | Memory policy for builtin             | |mempol|                | :class:`PREALLOCATED` |
-|                           | writers.                              |                         |                       |
-|                           |                                       |                         |                       |
-+---------------------------+---------------------------------------+-------------------------+-----------------------+
-| |mutTries|                | Number of different ports             | ``UInt32``              | 100                   |
-|                           | to try if reader's physical port      |                         |                       |
-|                           | is already in use.                    |                         |                       |
-+---------------------------+---------------------------------------+-------------------------+-----------------------+
++--------------------------------+----------------------------------+-------------------------+-----------------------+
+| Name                           | Description                      | Values                  | Default               |
++================================+==================================+=========================+=======================+
+| ``<discovery_config>``         | This is the main tag where       |                         |                       |
+|                                | discovery-related settings can   | :ref:`discovery_config  |                       |
+|                                | be configured.                   | <dconf>`                |                       |
++--------------------------------+----------------------------------+-------------------------+-----------------------+
+| ``<avoid_builtin_multicast>``  | Ignores the multicast locators   | ``Boolean``             | :class:`true`         |
+|                                | received from the remote         |                         |                       |
+|                                | participants.                    |                         |                       |
++--------------------------------+----------------------------------+-------------------------+-----------------------+
```

Comment:
Talking to @MiguelCompany we think this should be rewritten to:
"Restricts metatraffic multicast traffic to PDP only".
This is because not only multicast locators are ignored when received from the remote participant, but also the local participant does not send them.